tests/conftest.py

```diff
@@ -0,0 +1,65 @@
+import os
+import platform
+import shutil
+from pathlib import Path
+from typing import Optional, Callable, Any
+
+import allure
+import pytest
+from _pytest.fixtures import SubRequest
+from selenium import webdriver
+from utilities.WebDriverFactory import WebDriverFactory
+
+driver = None
+
+ALLURE_ENVIRONMENT_PROPERTIES_FILE = 'environment.properties'
+ALLUREDIR_OPTION = '--alluredir'
+browser_name = None
+# URL = "https://www.flipkart.com/"
+URL = "http://the-internet.herokuapp.com/"
+
+
+def pytest_addoption(parser):
+    parser.addoption("--browser_name", action="store", default="chrome")
+
+
+@pytest.fixture(scope="class")
+def setup(request):
+    global driver
+    global browser_name
+    browser_name = request.config.getoption("browser_name")
+    wdf = WebDriverFactory()
+    if browser_name == "chrome":
+        driver = wdf.get_chrome_driver()
+    elif browser_name == "firefox":
+        # driver = webdriver.Firefox(executable_path="D:\\FirefoxDriver\\geckodriver.exe")
+        driver = webdriver.Firefox()
+    elif browser_name == "IE":
+        print("IE driver")
```

Comment:
Please add driver for IE also and remove "print" comment line command

Reply:
Will update it in next PR